device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,484 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
+        "components": [],
+        "modules": [],
+        "fans": [
+            {
+                "name": "Fan 1",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 2",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 3",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 4",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 5",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 6",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 7",
+                "direction": "intake",
+                "tolerance": 20
+            },
+            {
+                "name": "Fan 8",
+                "direction": "intake",
+                "tolerance": 20
+            }
+        ],
+        "fan_drawers": [
+            {
+                "name": "Fan drawer 1",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 2",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 3",
+                "num_fans": 2
+            },
+            {
+                "name": "Fan drawer 4",
+                "num_fans": 2
+            }
+        ],
+        "psus": [
+            {
+                "name": "PSU 1",
+                "num_fans": 1,
+                "powergood_status": "true",
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2
+            },
+            {
+                "name": "PSU 2",
+                "num_fans": 1,
+                "powergood_status": "true",
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2
+            }
+        ],
+        "thermal_mgr": [
+		{
+		"FIXME":"need input from Mellanox for required static data"
+		}
+	],
+        "watchdog": [
+		{
+		"FIXME":"need input from Mellanox for required static data"
+		}
+	],
+        "eeprom": [
+		{
+		"FIXME":"need input from Mellanox for required static data"
+		}
+	],
+        "thermals": [
```

Comment:
the thermal sensors on Mellanox 2700 platform as below, the name of them can be found in the "Sensor" column, low threshold and low critical threshold is not available. The availability of the SFP thermal and threshold values depends on the capability of the cables, they are not fixed, other thermal sensors are fixed.
```
                  Sensor    Temperature    High TH    Low TH    Crit High TH    Crit Low TH    Warning          Timestamp
----------------------  -------------  ---------  --------  --------------  -------------  ---------  -----------------
                  ASIC           57.0      105.0       N/A           110.0            N/A      False  20200909 01:56:57
 Ambient Fan Side Temp           25.0        N/A       N/A             N/A            N/A      False  20200909 01:56:57
Ambient Port Side Temp           30.5        N/A       N/A             N/A            N/A      False  20200909 01:56:57
       CPU Core 0 Temp           26.0       87.0       N/A           105.0            N/A      False  20200909 01:56:57
       CPU Core 1 Temp           30.0       87.0       N/A           105.0            N/A      False  20200909 01:56:57
         CPU Pack Temp           30.0       87.0       N/A           105.0            N/A      False  20200909 01:56:57
            PSU-1 Temp           25.5      100.0       N/A             N/A            N/A      False  20200909 01:56:57
            PSU-2 Temp           23.5      100.0       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 1 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 2 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 3 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 4 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 5 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 6 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 7 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 8 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
    xSFP module 9 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 10 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 11 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 12 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 13 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 14 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 15 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 16 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 17 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 18 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 19 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 20 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 21 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 22 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 23 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 24 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 25 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 26 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 27 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 28 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 29 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 30 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 31 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
   xSFP module 32 Temp            N/A        N/A       N/A             N/A            N/A      False  20200909 01:56:57
```

Reply:
fixed, changed thermals according to this table